docs/operator-manual/upgrading/1.8-2.0.md

```diff
@@ -1,5 +1,13 @@
 # v1.8 to v2.0
 
+## Redis Upgraded to v6.2.1
+
+The bundled Redis version has been upgraded to v6.2.1.
+
+The Redis itself should be able to upgrade with no downtime, as well as Argo CD does use it as a persistent store.
```

Comment:
Do you mean "does not use it as persistent store" here?

Reply:
good catch. Thank you!